doc/api/inspector.md

```diff
@@ -49,6 +49,13 @@ and flow control has been passed to the debugger client.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+## inspector.waitForConnection()
+
+This method blocks until first client has connection, inspector should be open
```

Comment:
My reading of this patch is that this will wait until:
1. A connection was made on a WS endpoint.
2. Runtime.runIfWaitingForDebugger command was issued

runIfWaitingForDebugger should be mentioned here to reduce confusion